src/components/Bullet/index.ts

```diff
@@ -0,0 +1,3 @@
+import Bullet from './Bullet';
+
+export default Bullet;
```

Comment:
When you have a folder for a component and only that component inside that, I suggest having only index.tsx and moving all the code inside that file.